bin/build.sh

```diff
@@ -126,7 +97,10 @@ info "Built server ${CALABASH_ANDROID_SERVER_VERSION} with command:"
 
 shell "${CMD}"
 
-info "using ${ANDROID_TOOLS_DIR}"
-
 info "android:targetSdkVersion=${SERVER_API_LEVEL}"
 info "   android:minSdkVersion=${SERVER_MIN_VERSION}"
+
+zip -d "${APK_PATH}" META-INF/\*
+zip -d "${APK_PATH}" AndroidManifest.xml
```

Comment:
Can you add a `-q` to zip to make it shut up :-) And then `echo "Stripping signature and manifest from server apk..."` instead.

Reply:
done